io.catenax.week_based_capacity_group/2.0.0/WeekBasedCapacityGroup.ttl

```diff
@@ -0,0 +1,357 @@
+   samm:preferredName "Business Partner Number Legal Entity"@en ;
+   samm:description "BPNL represents the legal entity of an organization participating in Catena-X."@en ;
+   samm:dataType xsd:string .
+
+:BPNLRegularExpression a samm-c:RegularExpressionConstraint ;
+   samm:preferredName "BPNL Regular Expression"@en ;
+   samm:description "The provided regular expression ensures that the BPNL is composed of prefix 'BPNL', 10 digits and two uppercase letters."@en ;
+   samm:value "^BPNL[0-9]{8}[a-zA-Z0-9]{4}$" .
+
+:LinkedDemandSeries a samm:Entity ;
+   samm:preferredName "Linked Demand Series"@en ;
+   samm:description "Encapsulates information used to reference a specific demand series."@en ;
+   samm:properties ( :materialNumberCustomer [ samm:property :materialNumberSupplier; samm:optional true ] :customerLocation :demandCategory [ samm:property :loadFactor; samm:optional true ] ) .
+
+:Capacity a samm:Entity ;
+   samm:preferredName "Capacity"@en ;
+   samm:description "A supplier's realistically planned output per calendar week and material for a specific customer in a specific unit of measure, considering all positive or negative impacts on this capacity."@en ;
+   samm:properties ( :calendarWeek :actualCapacity :maximumCapacity [ samm:property :deltaProductionResult; samm:optional true ] ) .
+
+:materialNumberCustomer a samm:Property ;
+   samm:preferredName "Customer Material Number"@en ;
+   samm:description "Material identifier as assigned by customer. This material number identifies the material (as planned) in the customer's database."@en ;
+   samm:characteristic :MaterialNumber ;
+   samm:exampleValue "MNR-7307-AU340474.002" .
+
+:materialNumberSupplier a samm:Property ;
+   samm:preferredName "Supplier Material Number"@en ;
+   samm:description "Material identifier as assigned by supplier. This material number identifies the material (as planned) in the supplier's database. Used as informational field only, not for assigning demand series to capacity groups."@en ;
+   samm:characteristic :MaterialNumber ;
+   samm:exampleValue "MNR-8101-ID146955.001" .
+
+:customerLocation a samm:Property ;
+   samm:preferredName "Customer Location"@en ;
+   samm:description "The BPNS number of the site at which the customer needs the specified material for the referenced demand series."@en ;
+   samm:characteristic :BPNSTrait ;
+   samm:exampleValue "BPNS8888888888XX" .
+
+:demandCategory a samm:Property ;
+   samm:preferredName "Demand Category"@en ;
+   samm:description "Type of demand of the referenced demand series."@en ;
+   samm:characteristic :DemandCategory .
+
+:loadFactor a samm:Property ;
+   samm:preferredName "Load Factor"@en ;
+   samm:description "If existent, the load factor indicates that this material of the material demand collection takes a factor more or less intensity to produce."@en ;
+   samm:characteristic :LoadFactorCharacteristic ;
+   samm:exampleValue "3.5"^^xsd:decimal .
+
+:calendarWeek a samm:Property ;
```

Comment:
Should the naming be harmonized to pointInTime to better match the MaterialDemand model (https://github.com/eclipse-tractusx/sldt-semantic-models/pull/404/files#diff-94c912afcf6315943a92d314a8551725fcf2f552d6082837ae0021d043f69835R335)?

The IdBasedComment Model (https://github.com/eclipse-tractusx/sldt-semantic-models/pull/405/files#diff-8294717bc9be9d276f548ae9f83e2831503589ef17d4d5bf6f1463cdb397d00eR87) also refers to this property (and the pointInTime property of Material Demand), but names this concept a "Reference Date" -- which is a different name once again --, which in the description is called a "point in time".

To ease implementation, we should strive for a consistent naming with the new models.

Reply:
please check most recent RELEASE_NOTES.md and .ttl of PR